### ISSUE TITLE
Reproducing the secure demo, SECURITY ERROR

### ISSUE THREAD

**Issue by vmayoral** [mikaelarguedas/tb3_demo#29]
@ruffsl and @mikaelarguedas, 

I gave it a try to the secure demo and ended up with the following setup:
- Host machine: OS X 10.14.3
- Docker: Docker version 19.03.2, build 6a30dfc

Launched demo as follows:
```bash
rocker rosswg/turtlebot3_demo:latest "byobu -f configs/secure.conf attach"
```
(image pre-built from https://github.com/vmayoral/turtlebot3_demo fork)
and got something like:
<img width="715" alt="imagen" src="https://user-images.githubusercontent.com/1375246/67505816-3a91d180-f68c-11e9-883f-96bd5e5dfc95.png">

None of the tabs allow me to introspect ROS 2 with the ros2cli tools and I get a similar output:
```bash
root@29e19fd1a187:/opt# ros2 node list
Unknown error creating node: SECURITY ERROR: unable to find a folder matching the node name in /opt/tb3_overlay_ws/../keystore/.Lookup strategy: MATCH_PREFIX, at /tmp/binarydeb/ros-dashing-rcl-0.7.7/src/rcl/security_directory.c:252
```

**Comment by vmayoral** [comment]
I reproduced the same in a Linux host and obtained the same result.

**Comment by vmayoral** [comment]
As opposed to this:
```bash
rocker rosswg/turtlebot3_demo:roscon19 "byobu -f configs/secure.conf attach"
```

which seems to work just fine. Seems like my contributions at https://github.com/ros-swg/turtlebot3_demo/pull/19 are breaking things somehow.

**Comment by mikaelarguedas** [comment]
Can you make sure you're on top of master?
I changed the way we generated the ros2cli permissions recently, maybe it's related ..

I found a few occurences where I forgot to update the generate_artifacts invocations :S  can you give https://github.com/ros-swg/turtlebot3_demo/pull/30 a try or check if running the following command fixes it:
```
   ros2 security generate_artifacts -k keystore \
      -p policies/tb3_gazebo_policy.xml \
      -n /_ros2cli
```

**Comment by vmayoral** [comment]
Thanks for the quick response @mikaelarguedas,

```bash
 ros2 security generate_artifacts -k keystore \
      -p policies/tb3_gazebo_policy.xml \
      -n /_ros2cli
```

Fixed it. I can now use ros2cli from new tabs.

I tested it from my fork, let me re-try on top of master.

**Comment by vmayoral** [comment]
Nah, same issue in master @mikaelarguedas. Do you want me to include that command above or would you do it yourself?

**Comment by mikaelarguedas** [comment]
I'll just merge https://github.com/ros-swg/turtlebot3_demo/pull/30 if this fixes the issue for you

**Comment by mikaelarguedas** [comment]
This should be addressed by #30

**Comment by vmayoral** [comment]
Thanks @mikaelarguedas, closing here.
